googlesearch/__init__.py

```diff
@@ -141,14 +143,13 @@ def get_tbs(from_date, to_date):
     :rtype: str
     :return: Dates encoded in tbs format.
     """
-    from_date = from_date.strftime('%m/%d/%Y')
-    to_date = to_date.strftime('%m/%d/%Y')
+    from_date, to_date = from_date.strftime('%m/%d/%Y'), to_date.strftime('%m/%d/%Y')
     return 'cdr:1,cd_min:%(from_date)s,cd_max:%(to_date)s' % vars()
 
 
 # Request the given URL and return the response page, using the cookie jar.
 # If the cookie jar is inaccessible, the errors are ignored.
-def get_page(url, user_agent=None, verify_ssl=True):
+def get_page(url:str, user_agent:str=None, verify_ssl:bool=True) -> str:
```

Comment:
i see you have ran this into a linter, but this violates PEP-8. there should be a whitespace after the colons, and before/after the ='s